tests/integrationtests/tst_perfparser.cpp

```diff
@@ -159,9 +160,21 @@ class TestPerfParser : public QObject
 private slots:
     void initTestCase()
     {
-        if (!PerfRecord::isPerfInstalled()) {
+        RecordHost host;
+        host.setHost(QStringLiteral("localhost"));
+        QSignalSpy capabilitiesSpy(&host, &RecordHost::perfCapabilitiesChanged);
+        QSignalSpy installedSpy(&host, &RecordHost::isPerfInstalledChanged);
+        Q_ASSERT(installedSpy.wait());
+        if (!host.isPerfInstalled()) {
             QSKIP("perf is not available, cannot run integration tests.");
         }
+
+        if (capabilitiesSpy.count() == 0) {
+            Q_ASSERT(capabilitiesSpy.wait());
+        }
+        m_capabilities = host.perfCapabilities();
+
+        qDebug() << m_capabilities.canCompress << m_capabilities.canUseAio;
```

Comment:
qInfo, if you always want to show this

Reply:
no I just forgot to remove printf debugging